asmcomp/cmmgen.ml

```diff
@@ -138,6 +138,12 @@ let int_const dbg n =
   else Cconst_natint
           (Nativeint.add (Nativeint.shift_left (Nativeint.of_int n) 1) 1n, dbg)
 
+let natint_const dbg n =
```

Comment:
I find this naming is somewhat confusing, since the corresponding name for ints, `int_const` additionally tags the integer given, while this function does not. Maybe something like `cconst_natint` would better reflect the difference of semantics ?